### ISSUE TITLE
How to Freeze Graph in single .pb file with  Tensorflow.Net

### ISSUE THREAD

**Issue by zj2050** [SciSharp/TensorFlow.NET#833]
when train finished , i wanted to save graph in single .pb file , the save code like 
`
public void SaveModel(Session sess)
        {
            //save checkpoint of model using train.Saver.
            var saver = tf.train.Saver();
            var save_path = saver.save(sess, ".resources/logistic_regression/model.ckpt");
            print("Finish save checkpoint");
            //this will only save the graph but variables will not be saved.
            tf.train.write_graph(sess.graph, ".resources/logistic_regression", "model.pbtxt", as_text:true);
            print("Finish write graph");
            //this will freeze graph (graph + parametters)

            FreezeGraph.freeze_graph(input_graph: ".resources/logistic_regression/model.pbtxt",
                              input_saver: "",
                              input_binary: false,
                              input_checkpoint: ".resources/logistic_regression/model.ckpt",
                              output_node_names: "Softmax",
                              restore_op_name: "save/restore_all",
                              filename_tensor_name: "save/Const:0",
                              output_graph: ".resources/logistic_regression/model.pb",
                              clear_devices: true,
                              initializer_nodes: "");
            print("Finish freeze graph");
        }
`
but output only checkpoint file, text graph WITHOUT .pb file
![102732684-b6fe6e00-437e-11eb-8881-415ab7d11595](https://user-images.githubusercontent.com/4464947/128652843-b261a604-a0f5-43c1-8b13-a69be44cc416.jpg)

#702 

How to save the model as .pb file ?
Thank you very much.

**Comment by Oceania2018** [comment]
Not implemented yet.

**Comment by Muhammad-Talha-MT** [comment]
@Oceania2018 Please guide,  how to perform prediction in this case?

**Comment by Oceania2018** [comment]
@Muhammad-Talha-MT will upload sample code soon.

**Comment by zj2050** [comment]
> @Muhammad-Talha-MT will upload sample code soon.

Hi , when will upload new sample?

**Comment by Oceania2018** [comment]
Please check this [example](https://github.com/SciSharp/SciSharp-Stack-Examples/blob/master/src/TensorFlowNET.Examples/ImageProcessing/DigitRecognitionCNN.cs).
